lib/evaluate.js

```diff
@@ -41,17 +44,64 @@ export const evaluate = async ({
   // Clean up
   delete rounds[roundIndex]
 
+  // Calculate aggregates per fraud detection outcome
+  const assessments = {
+    OK: 0,
+    INVALID_TASK: 0
+  }
+  for (const m of measurements) {
+    assessments[m.fraudAssessment] = (assessments[m.fraudAssessment] ?? 0) + 1
+  }
+
   record('evaluate', point => {
     point.intField('round_index', roundIndex)
     point.intField('total_participants', Object.keys(participants).length)
     point.intField('total_measurements', measurements.length)
     point.intField('honest_measurements', honestMeasurements.length)
     point.intField('set_scores_duration_ms', setScoresDuration)
+
+    for (const a in assessments) {
+      point.intField(`measurements_${a}`, assessments[a])
+    }
```

Comment:
```suggestion
    for (const [type, count] of Object.entries(assessments)) {
      point.intField(`measurements_${type}`, count)
    }
```

To me this is simpler and more explicit